packages/shared/src/modules/activity-logs/libs/enums/activity-logs-api-path.enum.ts

```diff
@@ -0,0 +1,6 @@
+const ActivityLogsApiPath = {
+	ROOT: "/",
+	SAVE: "/save",
```

Comment:
```suggestion
```

Why do we need this save at all? Can't we post to ROOT?

Reply:
We have a different type of authorization for the POST method. So I add SAVE, so I can define this as a white route for plugin.

Reply:
Oh, I see. Just seems odd. Not really sure if it's needed or we can just stay with this save but we could add allowed methods to the white routes and make some minor adjustments to the authorization plugin for this. However, it's best to discuss this with coaches to avoid potential unnecessary work.

Reply:
@liza-veis  take a look please, i also do not like the name of this route